### PR TITLE
Add different new hooks for mini cart and cart

### LINKED ISSUE #23545: mini cart hooks
Hello woocommerce devs, I checked the cart/mini-cart.php and cart/cart.php and noticed that some hooks could be added inside to make the content more customizable.
1) You have woocommerce_widget_shopping_cart_before_buttons hook but you don't have the same after it something like woocommerce_widget_shopping_cart_after_buttons for wrapping some (added) content in one specific block element. Can it be implemented as a new hook ?
2) Can you guys add an action for woocommerce-mini-cart__total content or maybe whole paragraph with it's content inside :). It is 'chained' with the prev. task. (I want to have one block with subtotal and buttons in it, for now there is two blocks).
3) Cart/cart.php has woocommerce_cart_collaterals hook. Before it there is a form with woocommerce_before_cart_table and woocommerce_after_cart_table hooks inside. The question is - can you please, add the woocommerce_before_cart_collaterals and woocommerce_after_cart_collaterals hooks. It will help users who have two columns layout  ( there is a lot of those who do this way ) to add some content just after form. For this time we can add some content before collaterals with 'woocommerce_after_cart_table' hook but it's wrapped in form, which is not as good. (Ex. loader on refreshing is positioned according to all this form ). Thanks for support 👍

### PR DESCRIPTION
**Pull request by Mahjouba91**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #23545 .

### Changelog entry

New template hooks

cart/cart.php:
> woocommerce_before_cart_collaterals 

cart/mini-cart.php
> woocommerce_widget_shopping_cart_total
> woocommerce_widget_shopping_cart_after_buttons

### CODE REVIEW

**Comment by Mahjouba91** [comment]
There is no need to add a woocommerce_after_cart_collaterals since there is already a woocommerce_after_cart that can be used.

File: templates/cart/cart.php [hunk 1]
```diff
@@ -152,6 +152,8 @@
 	<?php do_action( 'woocommerce_after_cart_table' ); ?>
 </form>
 
+<?php do_action( 'woocommerce_before_cart_collaterals' ); ?>
```
Comment (claudiosanches) [inline templates/cart/cart.php]: This requires to bump this template version to 3.7.

File: templates/cart/mini-cart.php [hunk 2]
```diff
@@ -67,12 +67,14 @@
...
 	</ul>
 
-	<p class="woocommerce-mini-cart__total total"><strong><?php _e( 'Subtotal', 'woocommerce' ); ?>:</strong> <?php echo WC()->cart->get_cart_subtotal(); ?></p>
+	<p class="woocommerce-mini-cart__total total"><?php do_action( 'woocommerce_widget_shopping_cart_total' ); ?></p>
 
 	<?php do_action( 'woocommerce_widget_shopping_cart_before_buttons' ); ?>
 
 	<p class="woocommerce-mini-cart__buttons buttons"><?php do_action( 'woocommerce_widget_shopping_cart_buttons' ); ?></p>
 
+	<?php do_action( 'woocommerce_widget_shopping_cart_after_buttons' ); ?>
```
Comment (claudiosanches) [inline templates/cart/mini-cart.php]: Also need to increase this template version.

File: templates/cart/mini-cart.php [hunk 3]
```diff
@@ -67,12 +67,14 @@
 		?>
 	</ul>
 
-	<p class="woocommerce-mini-cart__total total"><strong><?php _e( 'Subtotal', 'woocommerce' ); ?>:</strong> <?php echo WC()->cart->get_cart_subtotal(); ?></p>
+	<p class="woocommerce-mini-cart__total total"><?php do_action( 'woocommerce_widget_shopping_cart_total' ); ?></p>
```
Comment (claudiosanches) [inline templates/cart/mini-cart.php]: Missing docblock here saying about calling `woocommerce_widget_shopping_cart_subtotal` and the priority, like we does for another actions like:

```php
<p class="woocommerce-mini-cart__total total"><?php 
    /**
     * woocommerce_widget_shopping_cart_total hook.
     *
     * @hooked woocommerce_widget_shopping_cart_subtotal - 10
     */
    do_action( 'woocommerce_widget_shopping_cart_total' );
?></p>
```

**Review by claudiosanches — CHANGES_REQUESTED** [review]
I just reviewed and there's a few things to improve, but only small things, in general looks good.

Thanks for your help!

**Comment by kloon** [comment]
@Mahjouba91 just checking if you saw @claudiosanches comments? If you can implement that we can get this reviewed and merged for the 3.7 release. Thanks!

**Comment by kloon** [comment]
Closing in favor of #24145

**Pull request closed without merging**